testcrate/src/lib.rs

```diff
@@ -20,14 +25,27 @@ pub fn version() -> (i32, i32, i32) {
     (major, minor, patch)
 }
 
+#[cfg(feature = "libsodium")]
+pub fn sodium_version() -> &'static CStr {
+    unsafe { CStr::from_ptr(sodium_version_string()) }
+}
+
 #[cfg(test)]
 mod test {
-    use super::version;
+    use super::*;
 
     #[test]
     fn version_works() {
         let version = version();
         println!("{:?}", version);
-        assert_eq!(version, (4, 3, 2));
+        assert_eq!(version, (4, 3, 4));
+    }
+
+    #[test]
+    #[cfg(feature = "libsodium")]
+    fn sodium_version_works() {
+        let version = sodium_version();
+        println!("{:?}", version.to_str().unwrap());
+        assert_eq!(version.to_str().unwrap(), "1.0.18");
```

Comment:
Just thought about this, this is gonna break because of semver when the libsodium version increases. It would be better instead to: `assert!(version.to_str().unwrap().starts_with("1.0"));`